gm4_bookshelf_inspector/beet.yaml

```diff
@@ -0,0 +1,23 @@
+id: gm4_bookshelf_inspector
+name: Bookshelf Inspector
+version: 1.0.X
+
+data_pack:
+  load: .
+
+pipeline:
+  - gm4.plugins.extend.module
+
+meta:
+  gm4:
+    versioning:
+      schedule_loops: [main]
+    website:
+      description: See what's in your Chiseled Bookshelves without having to take all the books out!
+      recommended: []
+      notes: []
```

Comment:
Since this only works on 1.20.2, there should be a note here that shows on the website